functests/test_suite_test.go

```diff
@@ -10,6 +10,11 @@ import (
 	. "github.com/onsi/ginkgo"
 	"github.com/onsi/ginkgo/reporters"
 	. "github.com/onsi/gomega"
+<<<<<<< Updated upstream
```

Comment:
@fedepaol I guess this is the place gofmt is not happy with ;)

Reply:
Yes I saw it, thanks.
Wonder how did I miss it